src/style.css

```diff
@@ -1,3 +1,112 @@
-body {
+.roboto-regular {
+  font-family: Roboto, serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  background: gray;
+}
+
+body {
+  font-family: Roboto, serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  background: #fff;
+  padding: 0 20px;
+}
+
+/* Logo styling */
+.moyoLogo {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 40px;
+  height: 40px;
+  margin-left: 30px;
+}
+
+.nav-links {
+  transform: translateX(-30px);
+  height: 60px;
+  margin-left: auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav-links ul {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  list-style: none;
+}
+
+.nav-links li + li {
+  margin-left: 20px;
+}
+
+.nav-item {
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+  text-decoration: none;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.nav-item:hover {
+  color: rgba(0, 172, 220, 1);
+}
+
+.is-active {
+  position: relative;
+  display: flex;
+  color: rgba(0, 172, 220, 1);
+  align-items: center;
+  background: white;
+}
+
+a.is-active {
+  height: 15px;
+  display: flex;
+  align-items: center;
+}
+
+.is-active::after {
+  content: '   ';
+  width: 100%;
+  height: 4px;
+  background-color: rgba(0, 172, 220, 1);
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  top: 33px;
+  border-radius: 8px;
+}
+
+@media (max-width: 1200px) {
+  html {
+    width: 1200px;
+  }
+}
+
+@media (max-width: 1024px) {
+  html {
+    width: 1024px;
```

Comment:
Similar to the previous comment, setting a fixed width for the 'html' element is not ideal for responsive design. It's better to use flexible layouts that adapt to different screen sizes.